Gemfile

```diff
@@ -59,3 +61,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem "hotwire-rails", "~> 0.1.3"
```

Comment:
Let's add `gem "stimulus-rails"` under line 15 instead of this gem. "hotwire-rails" basically bundles together "stimulus-rails" and "turbo-rails". Since they are basically standalone gems, I'd love to install them separately.